package/toltec-base/package

```diff
@@ -31,20 +31,23 @@ configure() {
         /opt/etc/profile
     echo "Disabling automatic update"
     disable-unit update-engine.service
-    if [[ "$arch" == "rm1" ]] && ! is-masked sys-subsystem-net-devices-usb1.device; then
+    if [[ "$arch" == "rm1" ]]; then
         echo "Disabling usb1 network device to avoid long boots"
-        systemctl mask sys-subsystem-net-devices-usb1.device
-    elif [[ "$arch" == "rm2" ]] && is-masked sys-subsystem-net-devices-usb1.device; then
+        ! systemctl is-masked sys-subsystem-net-devices-usb1.device && systemctl mask sys-subsystem-net-devices-usb1.device
+        ! systemctl is-masked busybox-ifplugd@usb1.service && systemctl mask busybox-ifplugd@usb1.service
```

Comment:
```suggestion
        if ! is-masked sys-subsystem-net-devices-usb1.device; then
            systemctl mask sys-subsystem-net-devices-usb1.device
        fi
        if ! is-masked busybox-ifplugd@usb1.service; then
            systemctl mask busybox-ifplugd@usb1.service
        fi
```
This will cause installation to fail if one of these units is already masked due to having a non-zero return code, and appending a `|| true` would keep it from failing if the mask command failed.